doc/userguide/configuration/multi-tenant.rst

```diff
@@ -195,26 +260,53 @@ Live traffic mode
 
 Multi-tenancy supports both VLAN and devices with live traffic.
 
-In the master configuration yaml file, specify ``device`` or ``vlan`` for the ``selector`` setting.
+In the master configuration yaml file, specify ``device``, ``vlan`` or ``vlan-tuple`` for the ``selector`` setting.
 
 Registration
 ~~~~~~~~~~~~
 
 Tenants can be mapped to vlan ids.
 
-``register-tenant-handler <tenant id> vlan <vlan id>``
+
+Examples using the ``vlan`` selector:
+
+::
+
+  register-tenant-handler <tenant id> vlan <vlan id>
 
 ::
 
   register-tenant-handler 1 vlan 1000
 
-``unregister-tenant-handler <tenant id> vlan <vlan id>``
+::
+
+  unregister-tenant-handler <tenant id> vlan <vlan id>
 
 ::
 
   unregister-tenant-handler 4 vlan 1111
   unregister-tenant-handler 1 vlan 1000
 
+
+Examples using the ``vlan-tuple`` selector:
+
+::
+
+  register-tenant-handler <tenant id> vlan-tuple <vlan outer id> <vlan inner id>
```

Comment:
should we use the tuple notation from the yaml here?
```
register-tenant-handler 1 vlan-tuple "[1000, 100, 3]"
```
not sure if it makes sense

Reply:
I don't know if that makes sense. None of the existing commands accept/require a yaml format.

Reply:
Not supportet yet, but at least for the syntax, simply:
```
register-tenant-handler 1 vlan-tuple 2 3 4 5 6
```

Reply:
Ok. @jlucovsky can you make sure to add support for that in a new version of the PR?

Reply:
Supporting a variable number of parameters is difficult without moving to a list format:
E.g.,
This format: `register-tenant-handler 1 vlan-tuple [2, 3, 4]` can be supported well but means that registering or unregistering a handler for `vlan` would also change: `register-tenant-handler 1 vlan [2]`.

I can support this in a backward compatible way so legacy `suricatasc` clients can use `register-tenant-handler 1 vlan 2` (legacy means pre-8.0 clients or clients who are unaware of `vlan-tuple`).

Reply:
I don't really see how `register-tenant-handler 1 vlan-tuple [2, 3, 4]` is much different than `register-tenant-handler 1 vlan-tuple 2 3 4`? Should be able to parse them to the same representation without ambiguity?

Reply:
The complication comes from `spec.py`, I think.

Reply:
The current format is fine, and works.  To support 3 we can add another optional arg, and so on up to some max. A bit tedious, but works fine.  Alternatively we could update spec.py with a new data type that parses the rest of the args into an array. But the end result is the same.

Reply:
> I'll make a new PR that supports `register-tenant-handler 1 vlan-tuple [n [,m [,z]]]`

But only to the level Suricata supports internally right?